core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -143,6 +143,21 @@ trait WorkspaceSupport {
     }
   }
 
+  def withWorkspaceBucketRegionCheck[T](bucketRegion: Option[String])(op: => Future[T]): Future[T] = {
+    bucketRegion match {
+      case Some(region) =>
+        // if the user specifies a region for the workspace bucket, it must be in the proper format for a single region
+        val singleRegionPattern = "[A-Za-z]+-[A-Za-z]+[0-9]+"
+        if (region.matches(singleRegionPattern)) op
+        else {
+          val err = ErrorReport(statusCode = StatusCodes.BadRequest, message = s"Workspace Bucket Location should be " +
+            s"of format: $singleRegionPattern.")
```

Comment:
I think this might be more user friendly if we say `"Bucket location must be a single (not multi-) region in the google regions list (https://cloud.google.com/compute/docs/regions-zones)"` (ie rather than expecting them to interpret the regex)

Reply:
Because the check doesn't actually compare the region submitted to the list of regions from Google, we're hesitant to put that in the error message.  How about `"Workspace bucket location must be a single (not multi-) region of format: [A-Za-z]+-[A-Za-z]+[0-9]+"`?